test/minters/redeem/ZoraCreatorRedeemMinterStrategy.t.sol

```diff
@@ -14,1485 +14,1486 @@ import {ICreatorRoyaltiesControl} from "../../../src/interfaces/ICreatorRoyaltie
-
-    function test_ResetSaleAlwaysReverts() external {
-        vm.prank(address(target));
-        vm.expectRevert(abi.encodeWithSignature("MustCallClearRedeem()"));
-        redeemMinter.resetSale(uint256(1));
-    }
-
-    function test_ClearRedeem() external {
-        vm.startPrank(admin);
-
-        address tokenRecipient = address(322);
-        vm.deal(tokenRecipient, 20 ether);
-
-        ERC721PresetMinterPauserAutoId burnToken = new ERC721PresetMinterPauserAutoId("Test token", "TEST", "https://zora.co/testing/token.json");
-        burnToken.mint(address(tokenRecipient));
-        burnToken.mint(address(tokenRecipient));
-
-        ZoraCreatorRedeemMinterStrategy.MintToken memory mintToken = ZoraCreatorRedeemMinterStrategy.MintToken({
-            tokenContract: address(target),
-            tokenId: newTokenId,
-            amount: 10,
-            tokenType: ZoraCreatorRedeemMinterStrategy.TokenType.ERC1155
-        });
-        ZoraCreatorRedeemMinterStrategy.RedeemInstruction[] memory instructions = new ZoraCreatorRedeemMinterStrategy.RedeemInstruction[](1);
-        instructions[0] = ZoraCreatorRedeemMinterStrategy.RedeemInstruction({
-            tokenType: ZoraCreatorRedeemMinterStrategy.TokenType.ERC721,
-            amount: 2,
-            tokenIdStart: 0,
-            tokenIdEnd: 1,
-            tokenContract: address(burnToken),
-            transferRecipient: address(0),
-            burnFunction: bytes4(keccak256(bytes("burn(uint256)")))
-        });
-        ZoraCreatorRedeemMinterStrategy.RedeemInstructions memory redeemInstructions = ZoraCreatorRedeemMinterStrategy.RedeemInstructions({
-            mintToken: mintToken,
-            instructions: instructions,
-            saleStart: 0,
-            saleEnd: type(uint64).max,
-            ethAmount: 1 ether,
-            ethRecipient: address(0)
-        });
-
-        target.callSale(newTokenId, redeemMinter, abi.encodeWithSelector(ZoraCreatorRedeemMinterStrategy.setRedeem.selector, redeemInstructions));
-        bytes32[] memory hashes = new bytes32[](1);
-        hashes[0] = keccak256(abi.encode(redeemInstructions));
-        vm.expectEmit(true, false, false, true);
-        emit RedeemsCleared(address(target), hashes);
-        target.callSale(newTokenId, redeemMinter, abi.encodeWithSelector(ZoraCreatorRedeemMinterStrategy.clearRedeem.selector, hashes));
-        vm.stopPrank();
-
-        vm.startPrank(tokenRecipient);
-        burnToken.setApprovalForAll(address(redeemMinter), true);
-        uint256[][] memory tokenIds = new uint256[][](1);
-        tokenIds[0] = new uint256[](2);
-        tokenIds[0][0] = 0;
-        tokenIds[0][1] = 1;
-        uint256[][] memory amounts = new uint256[][](1);
-        vm.expectRevert(abi.encodeWithSignature("RedeemInstructionNotAllowed()"));
-        target.mint{value: 1 ether}(redeemMinter, newTokenId, 10, abi.encode(redeemInstructions, tokenIds, amounts));
-    }
-}
+/// @notice v1.4.1 will not support burn to redeem
```

Comment:
what's the reason for b2r not supported in 1.4.1?

Reply:
the assumption of first arg is uint256 tokenId breaks compatibility with the minter + product is deprecating support for b2r

Reply:
got it - thoughts on doing that in a separate PR that actually removes the contracts and relevant code?